tasks/system_probe.py

```diff
@@ -188,50 +152,51 @@ def test(
     ctx,
     packages=TEST_PACKAGES,
     skip_object_files=False,
-    bundle_ebpf=True,
+    bundle_ebpf=False,
     output_path=None,
     runtime_compiled=False,
     skip_linters=False,
```

Comment:
What do you think about making this `True`, but CI sets it to `False`? There is far too much output when running tests locally.

Reply:
I think folks are less likely to see the linter output if it only runs in CI. The "too much output" is somewhat intentional, because you should be motivated to reduce it.

Reply:
The opposite may also happen: i.e., too much output, so ignore it completely. I guess we can't make it a CI job failure yet, so maybe we can hold off until then.